src/components/UI/ErrorFallback.js

```diff
@@ -50,7 +50,11 @@ function ErrorFallback({ errors, children }) {
 }
 
 ErrorFallback.propTypes = {
-  errors: PropTypes.oneOfType([PropTypes.string, PropTypes.array]),
+  errors: PropTypes.oneOfType([
+    PropTypes.object,
+    PropTypes.string,
+    PropTypes.array,
+  ]),
   children: PropTypes.node,
```

Comment:
Sorry, I wasn't clear enough: Why do we want to pass the full object here? What do we get from this?